client/lib/plans/constants.js

```diff
@@ -686,7 +686,22 @@ export const PLANS_LIST = {
 		...getPlanEcommerceDetails(),
 		term: TERM_ANNUALLY,
 		getBillingTimeFrame: WPComGetBillingTimeframe,
-		availableFor: () => [],
+		availableFor: plan =>
```

Comment:
Does adding the lower plans `availableFor` enable upgrades and the length selector ?
We need the length selector but we also need to disable eCommerce upgrades until we are to upgrade Simple sites to Atomic V2.

Reply:
Ah I see! I just updated this PR - does that look better?